### PR TITLE
Svg

### LINKED ISSUE #22: Add draft map for Chapter 1
Embed inline as svg

### PR DESCRIPTION
**Pull request by ellenbechtel**
Before making a pull request
----------------------------
First . . .
- [x] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Cleaning up CSS for SVG of intro map
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Not complete, but wanted to get it out there for viewing. Still have to work on:
- removing/adding certain labels on certain screen sizes
- tweaking font size for certain screen sizes
- masking for labels inside the DRB so they aren't cluttered

-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/views/Visualization.vue [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 
 <script>
-  
+  import Importance from "/1-Importance/Importance"
+
+  export default {
+    name: "Vizualization",
+    components: {
+      Importance: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "nav"*/ "./components/Nav"),
+      Importance: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "importance"*/ "./1-Importance/Importance")
```
Comment (cnell-usgs) [inline src/views/Visualization.vue]: Should there be two `Importance` components?

**Review by cnell-usgs — APPROVED** [review]

**Review by hcorson-dosch-usgs — COMMENTED** [review]
Sweet! SO awesome to see this in here looking so crisp.

A few things that I think need tweaking (many of which I'm guessing you're already aware of) - these are not urgent though, so could be tackled later on. Also, my apologies b/c I am admittedly very nitpicky about maps:
* I think the circle markers for the cities need to be a bit larger
* I think I'd prefer a grey color for the markers over blue, since we're using blue to denote water
* Could you  make the labels for the 4 cities within the watershed a lighter grey than the labels for those outside of the watershed?
* It looks like the labels for the the cities within the watershed lost that 2px black stroke that was behind the labels. Is it possible to have that there w/ the svg? I think it would help the labels to be legible by hiding the rivers behind the labels.
* Could the labels for Chesapeake Bay and Long Island Sound be a bit darker than that for Delaware Bay? Or, conversely, could you lighten the Delaware Bay label a bit?
* Could you lighten the coastal buffers a tiiiiiny bit so that they are slightly more visible?
* The font in the scale bar looks to be defaulting to some serif font, rather than the sans-serif used elsewhere
* I think it would be worth seeing if you can get that glow around the watershed a bit brighter/wider, like you were mentioning yesterday
* Maybe try making the dotted state lines a bit lighter grey? And/or maybe up the stroke width a bit?
* The river labels do pop in white, which I think helps them be more legible, given how small the font size is, but maybe try a super light blue to see how that looks?
* The tracking seems to have changed a bit for a few of the state labels, notably New York (much less tracking) and Delaware (label now abuts water) and Chesapeake Bay -- maybe that was tied to the font change?
* I don't love this font, mostly b/c there is no 'light' font weight, which I had used for the state labels - how much of a pain would it be to try a different sans-serif font with a light font-weight in addition to regular?
* It's very subtle and I don't think it's a problem, but on Chrome the map flickers slightly when you scroll, which is odd. No idea what might be causing that.
* And, lastly -- I assume we may just leave this to later, since we'll have lots to tweak in this department, but I think we will need to fine tune the map size for different screen sizes, and make sure the map isn't cut off on mobile

File: src/views/Visualization.vue (hunk 1 above)
Comment (ellenbechtel) [reply src/views/Visualization.vue]: Oh right - this component (Visualization.vue) isn't working.  Currently everything's getting loaded into App.vue. So this currently doesn't effect anything, but was sloppy on my part regardless.

**Comment by ellenbechtel** [comment]
Great @hcorson-dosch , these nitpicks are really useful!  I'll wrap that up and the few things I had on my list and then call it done. Probs by tomorrow's pitchroom.  Woohoo!

**Comment by hcorson-dosch-usgs** [comment]
Fixes #22

**Pull request merged**